script/build_frontend

```diff
@@ -60,6 +59,7 @@ gzip -f -n -k -9 \
     *.js \
     ./panels/*.html \
     ./translations/*.json \
+    ./translations/*/*.json \
```

Comment:
When working on #678 I realized that `script/build_frontend` shouldn't exist. It's like doing all these tasks that we should turn into gulp tasks. Gzipping a folder is easily done in gulp: [example](https://github.com/home-assistant/home-assistant-polymer/pull/678/files#diff-0f4211b4a498fe0ef4990a1a55e17ec1R38)

Reply:
Agree with that, but I'd suggest we migrate this in a separate PR.

Reply:
Agree